src/main/resources/application.properties

```diff
@@ -123,9 +123,9 @@ server.error.include-stacktrace=never
 nae.pdf.resources.outputFolder=storage
 nae.pdf.resources.outputDefaultName=generated_pdf.pdf
 nae.pdf.resources.templateResource=file:src/main/resources/pdfGenerator/template_pdf.pdf
-nae.pdf.resources.font-title-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
-nae.pdf.resources.font-label-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
-nae.pdf.resources.font-value-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Bold.ttf
+nae.pdf.resources.font-title-resource=file:src/main/resources/pdfGenerator/fonts/Roboto-Light.ttf
```

Comment:
is this just a change of the default font? Or should the bold variant not be supported at all? Judging by the fact the light font .ttf file was added in this PR, then the bold .ttf file should be removed from the project (assuming it is a part of it).

Reply:
This is a default font, the bold can be used as well, I can remove the unused fonts if you would like to.

Reply:
if the bold font is never referenced then it should be removed in my opinion